yarn.lock

```diff
@@ -2122,10 +2123,22 @@ __metadata:
   languageName: unknown
   linkType: soft
 
+"@jupyter/collaborative-drive@^3.0.0-beta.6, @jupyter/collaborative-drive@workspace:packages/collaborative-drive":
```

Comment:
This line looks different than for the other packages.

Reply:
Good catch, let's try to make it similar.

Reply:
So it's Yarn that adds `@jupyter/collaborative-drive@^3.0.0-beta.6`.